torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -688,6 +688,94 @@ graph(%a_quant, %negative_slope):
           %r = aten::leaky_relu_(%a_quant, %negative_slope)
           return (%r) )";
 
+  // IR pattern common to all ops with fixed quantization parameters for
+  // asymetric quantization
+  std::string asym_fixed_qparam_op_suffix = R"(
```

Comment:
can we also add some comments on the magic numbers here, even if we can't pull them out into constants?